README.md

```diff
@@ -99,60 +99,60 @@ You can get it from http://repo1.maven.org/maven2/org/glassfish/javax.json/1.0.4
 
 ```
 
-When the javax.json-1.0.4.jar, commons-codec-1.1.jar and maximo-restclient-0.1.jar is ready, add them to the java project as common reference libraries.
+When the javax.json-1.0.4.jar, commons-codec-1.1.jar, and maximo-restclient-0.1.jar files are ready, add them to the Java project as common reference libraries.
 	
 # III. Usage
 -----
 
-Maximo Resources (Object Structures) represent a graph of related Maximo objects (Mbo's) that provides an atomic view/object to create/update/delete/query the releated set of Mbos. 
+Maximo Resources, or object structures, represent a graph of related Maximo business objects (Mbos) that provides an atomic view/object to create, update, delete, and query the releated set of Mbos. 
```

Comment:
line 107
/ is particularly untranslateable and both humans and machines can misinterpret it. better to use whatever word works best. i 'm not sure which word belongs in place of the slash here.